README.md

```diff
@@ -865,6 +865,7 @@ This list is the result of Pull Requests, reviews, ideas and work done by 1100+
   * [Mailjet](https://www.mailjet.com/) — 6,000 emails/month free (200 emails daily sending limit)
   * [Mailnesia](https://mailnesia.com) - Free temporary/disposable email, which auto visit registration link.
   * [mailsac.com](https://mailsac.com) - Free API for temporary email testing, free public email hosting, outbound capture, email-to-slack/websocket/webhook (1,500 monthly API limit)
+  * [MailSlurp.com](https://www.mailslurp.com/) — Temporary/disposable email, fake SMTP server, webhooks, and sending. Free plan with 50 inboxes, 100 messages, no team members, webhooks, or custom domains.
```

Comment:
I cannot find a description of whats free on the pricing page, mind sharing a link?